sample-compose/app/src/test/java/co/nimblehq/sample/compose/ui/screens/home/HomeViewModelTest.kt

```diff
@@ -0,0 +1,84 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import app.cash.turbine.test
+import co.nimblehq.sample.compose.domain.model.Model
+import co.nimblehq.sample.compose.domain.usecase.UseCase
+import co.nimblehq.sample.compose.model.toUiModel
+import co.nimblehq.sample.compose.test.CoroutineTestRule
+import co.nimblehq.sample.compose.ui.AppDestination
+import co.nimblehq.sample.compose.util.DispatchersProvider
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.*
+import org.junit.*
+
+@ExperimentalCoroutinesApi
+class HomeViewModelTest {
+
+    @get:Rule
+    val coroutinesRule = CoroutineTestRule()
+
+    private val mockUseCase: UseCase = mockk()
+
+    private lateinit var viewModel: HomeViewModel
+
+    private val models = listOf(Model(1), Model(2), Model(3))
+
+    @Before
+    fun setUp() {
+        every { mockUseCase() } returns flowOf(models)
+
+        initViewModel()
+    }
+
+    @Test
+    fun `when loading models successfully, it shows the model list`() = runTest {
```

Comment:
For consistency
```suggestion
    fun `When loading models successfully, it shows the model list`() = runTest {
```

Reply:
Fixed all in part 2/2 https://github.com/nimblehq/android-templates/pull/399/commits/5f3f9c0d2d79480dafac6530e78426c800fe42e2